api/agent.go

```diff
@@ -322,15 +333,60 @@ func NewControllerWrapper(config *loadtest.Config, controllerConfig interface{},
 		}
 	}
 
+	var creds []string
+	f, err := os.Open(config.UsersConfiguration.UsersFilePath)
```

Comment:
Why don't we just return an error if `UsersFilePath` is not empty and `os.Open()` fails?

Reply:
We can if you want to. I thought of falling back to the default signup if the file open fails, with a warning message.

Reply:
This is a common problem. Personally, given the load-test is by nature a time consuming process, I'd prefer being precise and apply a "fail fast" approach. If a path was given in the config I'd expect it's needed. The user could miss the warning and think everything went okay since the load-test will run anyway, only to find out the wrong users logged in. It would be the user's fault of course but we can prevent it by being extra explicit.